src/trainer/evaluator.py

```diff
@@ -332,7 +332,7 @@ def get_state(partial_obs):
                 .to(device=self.device, dtype=torch.float32)
             )
 
-        max_ep_len = env.get_env().max_steps
+        max_ep_len = env.max_steps
```

Comment:
Are we sure we don't need this here anymore? I understand RecorderEnv inherits from FeedbackEnv now - just confused why we're still using env.get_env() elsewhere (lines 324, 348) 😅